app/components/form/fields/ComboboxField.tsx

```diff
@@ -38,22 +37,37 @@ export function ComboboxField<
   label = capitalize(name),
   required,
   onChange,
-  disabled,
+  allowArbitraryValues,
+  placeholder,
+  // Intent is to not show both a placeholder and a description, while still having good defaults; prefer a description to a placeholder
+  /*
+   * If description is provided, use it.
+   * If not, but a placeholder is provided, the default description should be undefined.
+   * If no placeholder is provided and arbitrary values are allowed, the default description should be 'Select an option or enter a custom value'.
+   * If no placeholder is provided and arbitrary values are not allowed, the default description should be 'Select an option'.
+   */
+  description = placeholder
+    ? undefined
+    : allowArbitraryValues
+      ? 'Select an option or enter a custom value'
+      : 'Select an option',
   ...props
 }: ComboboxFieldProps<TFieldValues, TName>) {
   const { field, fieldState } = useController({ name, control, rules: { required } })
   return (
     <div className="max-w-lg">
       <Combobox
-        isDisabled={disabled}
```

Comment:
Is this right? Seems like the mismatch between `disabled` and `isDisabled` may require this to be passed.

Reply:
Ah, I think I see what you did: you changed the prop on `ComboboxField` to `isDisabled` and let it get passed through as part of the props spread.

https://github.com/oxidecomputer/console/commit/045c58a90f66c3b5eab8d2d504272536b307a7ef

Reply:
Yeah, I realized that there was an existing `isDisabled` prop and it made sense to just use that.